Makefile

```diff
@@ -1,7 +1,24 @@
 ST2_VERSION ?= 3.4dev
 DOCKER_TAG ?= ${ST2_VERSION}
+RELEASE_TAG_REGEX := [^dev]$$
 SHELL := /bin/bash
 
+TAG_UPDATE_FLAG = $(shell ./determine_needed_tags.sh st2 ${ST2_VERSION})
+# supported values of the TAG_UPDATE_FLAG
+# 0 = no additional tags to be set
+# 1 = add the major.minor tag
+# 2 = add the tags major and major.minor
+# 3 = add the latest tag (for future use)
```

Comment:
Looks like "for future use" could be removed now as `latest` was impelemented

Reply:
Done.